nss-day-cohort-45/tabloidfullstack-giving-gillyflowers
# Ar restrict post edit

### Changes:
- protected Put method in `PostsController`
- removed edit/delete icons from `PostListCard`
- added error message if someone who isn't the author intrepidly navigates to the edit route and attempts to change a post
---
### Testing:
- fetch branch and run app and api
- log in
- navigate straight to the edit route of a post that you did not create (`http://localhost:3000/posts/edit/22` for example) and attempt to change and save it. you should see an error message and you should be directed back to the post detail view
- attempt to edit a post you _did_ create. it should still work.
